node/src/config/commands.js

```diff
@@ -9,7 +9,7 @@ const commands = {
     help: [
       {
         desc: 'Deploys an environment',
-        example: `$ env0 deploy -k <${API_KEY}> -s <${API_SECRET}> -o <${ORGANIZATION_ID}> -p <${PROJECT_ID}> -b <${BLUEPRINT_ID}> -e <${ENVIRONMENT_NAME}> -r [revision] -v [stage=dev]`
+        example: `$ env0 deploy -k <${API_KEY}> -s <${API_SECRET}> -o <${ORGANIZATION_ID}> -p <${PROJECT_ID}> -b <${BLUEPRINT_ID}> -e <${ENVIRONMENT_NAME}> -r [revision] -v [stage=dev] -u [tfvar=tfvalue]`
```

Comment:
Maybe also add example for regular environment variables here

Reply:
regular environment variables is still `-v [stage=dev]`

so not sure what you mean.